sdk/core/abort-controller/tsconfig.json

```diff
@@ -8,5 +8,13 @@
     "moduleResolution": "NodeNext",
     "rootDir": "."
   },
-  "include": ["./src/**/*.ts", "./src/**/*.mts", "./src/**/*.cts", "./test/**/*.ts"]
+  "include": [
+    "./src/**/*.ts",
+    "./src/**/*.mts",
+    "./src/**/*.cts",
+    "./test/**/*.ts",
+    "src/**/*.ts",
+    "test/**/*.ts",
```

Comment:
is silly eslint complaining about this line? We should remove `"./src/**/*.ts",` and `"./test/**/*.ts",` if adding these two.

Reply:
Yes, it is an ESLint issue.